src/test/java/seedu/realodex/testutil/TypicalPersons.java

```diff
@@ -87,6 +87,42 @@ public class TypicalPersons {
             .withEmail("hans@example.com").withAddress("chicago ave")
             .withTags("buyer").withHousingType("HDB").withBirthday("23Apr1983").build();
 
+    public static final Person HENRY = new PersonBuilder().withName("Henry Johnson").withPhone("9482541")
+            .withIncome("80000").withEmail("john@example.com").withAddress("5th avenue").withTags("seller")
+            .withHousingType("HDB").withBirthday("12Mar1995").build();
+
+    public static final Person ISABELLA = new PersonBuilder().withName("Isabella Lee").withPhone("9482682")
+            .withIncome("90000").withEmail("isabella@example.com").withAddress("main street").withTags("buyer")
+            .withHousingType("HDB").withBirthday("5Sep1990").build();
+
+    public static final Person JACKSON = new PersonBuilder().withName("Jackson Wang").withPhone("9482335")
+            .withIncome("100000").withEmail("jackson@example.com").withAddress("baker street").withTags("seller")
+            .withHousingType("HDB").withBirthday("8Jul1988").build();
+
+    public static final Person KATHERINE = new PersonBuilder().withName("Katherine Smith").withPhone("9482774")
+            .withIncome("110000").withEmail("katherine@example.com").withAddress("park avenue").withTags("buyer")
+            .withHousingType("HDB").withBirthday("16Nov1984").build();
+
+    public static final Person LUCAS = new PersonBuilder().withName("Lucas Brown").withPhone("9482993")
+            .withIncome("120000").withEmail("lucas@example.com").withAddress("ocean drive").withTags("seller")
+            .withHousingType("HDB").withBirthday("20Jan1980").build();
+
+    public static final Person MADELINE = new PersonBuilder().withName("Madeline Garcia").withPhone("9482662")
+            .withIncome("130000").withEmail("madeline@example.com").withAddress("wallace street").withTags("buyer")
+            .withHousingType("HDB").withBirthday("9Apr1976").build();
+
+    public static final Person NATHANIEL = new PersonBuilder().withName("Nathaniel Martinez").withPhone("9482891")
+            .withIncome("140000").withEmail("nathaniel@example.com").withAddress("elm street").withTags("seller")
+            .withHousingType("HDB").withBirthday("14Aug1972").build();
+
+    public static final Person OLIVIA = new PersonBuilder().withName("Olivia Taylor").withPhone("9482889")
+            .withIncome("150000").withEmail("olivia@example.com").withAddress("maple street").withTags("buyer")
+            .withHousingType("HDB").withBirthday("17Dec1968").build();
+
+
```

Comment:
did u add NEW PEOPLE

Reply:
My friend gpt did 😊

Reply:
I wanted to test sorting using another new list

Reply:
👍🏻